rust/content.md

```diff
@@ -30,6 +30,26 @@ $ docker build -t my-rust-app .
 $ docker run -it --rm --name my-running-app my-rust-app
 ```
 
+This creates an image that has all of the rust tooling for the image, which is 1.8gb. If you just want the compiled application:
+
+```dockerfile
+FROM rust:1.39 as builder
+WORKDIR /usr/src/myapp
+COPY . .
+RUN cargo install --path .
+
+FROM debian:buster-slim
+RUN apt-get update && apt-get install -y libssl-dev
```

Comment:
This could be just `libssl1.1`, couldn't it?  (It probably doesn't need the headers, etc at runtime.)

Reply:
Good catch. Made change.